index.js

```diff
@@ -85,41 +85,41 @@ NYC.prototype._createInstrumenter = function () {
 NYC.prototype._prepGlobPatterns = function (patterns) {
   if (!patterns) return patterns
 
-  var directories = []
-  patterns = _.map(patterns, function (pattern) {
+  var result = []
+
+  function add (pattern) {
+    if (result.indexOf(pattern) === -1) {
+      result.push(pattern)
+    }
+  }
+
+  patterns.forEach(function (pattern) {
     // Allow gitignore style of directory exclusion
-    if (!_.endsWith(pattern, '/**')) {
-      directories.push(pattern.replace(/\/$/, '').concat('/**'))
+    if (!endsWith(pattern, '/**')) {
+      add(pattern.replace(/\/$/, '').concat('/**'))
```

Comment:
(And yes you didn't write `concat()`. But since we're speading things up :stuck_out_tongue_winking_eye: )